examples/WebApi/Controllers/HandleFaxEventController.cs

```diff
@@ -0,0 +1,27 @@
+using Microsoft.AspNetCore.Mvc;
+using Sinch.Fax.Hooks;
+
+namespace WebApiExamples.Controllers
+{
+    [ApiController]
+    [Route("fax")]
+    public class HandleFaxEventController : ControllerBase
+    {
+        [HttpPost]
+        [Route("event")]
+        public IActionResult HandleEvent([FromBody] IFaxEvent faxEvent)
+        {
+            switch (faxEvent)
+            {
+                case IncomingFaxEvent incomingFaxEvent:
```

Comment:
As a SDK user, I think it would be interesting to see how an event is handled, both as application/json and as multipart/form-data. Can you extend this example?

Reply:
Yes, I see 2 of them (both valid for the 2 content types)
 - store an incoming fax
 - keep a record of sent faxes